util/test-utils/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+#![allow(warnings)]
+use tempfile::{NamedTempFile, TempDir};
+
+/// Create a temporary directory in the directory specified by the
+/// cargo-provided `OUT_DIR` environment variable.
+///
+/// # Panics
+///
+/// - If `OUT_DIR` doesn't exist
+/// - If [`TempDir::new_in`] fails to create the directory.
+pub fn tempdir() -> TempDir {
+    let out = std::env::var("OUT_DIR").expect("Missing environment variable OUT_DIR");
+    TempDir::new_in(&out).expect(&format!("Could not create temporary directory {}", &out))
```

Comment:
Why should `expect` be replaced? This seems to replicate what expect already does https://doc.rust-lang.org/stable/std/result/enum.Result.html#method.expect

Is this to avoid the unconditional `format!` call?

Reply:
Yes, exactly. There's a Clippy check that requires this. If it doesn't trigger for this change, I'm fine with the `expect` call as written.